lib/package.json

```diff
@@ -34,17 +34,22 @@
   "peerDependencies": {
     "classnames": "^2.2.5",
     "material-ui": "^1.0.0-beta.33",
-    "moment": "^2.19.2",
     "prop-types": "^15.6.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0"
   },
+  "externalDependencies": {
```

Comment:
didn't find any info about this `externalDependencies` key. Shouldn't `moment` and `date-fns` be in `peerDependencies`? I'm not sure if it's possible to mention `one of` there though...

Reply:
Its not possible, and good that you mention - I forgot to add error inside the utils file, that some of required library not installed

Reply:
What do you think about printing info about utils after package is installed? Like ['jss' does it for opencollective support](https://github.com/cssinjs/jss/blob/09f37523dad9622a5f25c69dffe4a4b2e7bee681/package.json#L111) ?